runchooks/runchooks.c

```diff
@@ -0,0 +1,68 @@
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <dlfcn.h>
+#include <unistd.h>
+#include <string.h>
+#include <strings.h>
+#include <libgen.h>
+
```

Comment:
Are dlfcn, strings and libgen really required?

Reply:
No. Removing.